ui/src/app/job-list/table/table.component.ts

```diff
@@ -41,6 +42,7 @@ export class JobsTableComponent implements OnInit {
   private mouseoverJob: QueryJobsResult;
 
   public displayFields: DisplayField[];
+  public bulkLabelFields;
```

Comment:
Please use a type:

```
...: Array<{
  default: string|null,
  field: DisplayField
}>;
```

After this change, please fix all accesses to use normal property name access instead of using strings. You should only need to use strings to access properties if the property name is actually dynamic (e.g. for labels).